lib/src/html/html_generator.dart

```diff
@@ -61,9 +65,9 @@ class HtmlGenerator extends Generator {
 
   @override
   Future generate(Package package, Directory out) {
-    return new HtmlGeneratorInstance(
-            _options, _templates, package, out, _onFileCreated)
-        .generate();
+    _instance = new HtmlGeneratorInstance(
```

Comment:
We're recording this in the object now? Should we document this, and fail if an `_instance` already exists (I have not looked at the call sites).